litmus-portal/frontend/src/views/CreateWorkflow/TuneWorkflow/WorkflowTable.tsx

```diff
@@ -191,16 +179,16 @@ const WorkflowTable = forwardRef(({ isCustom }: WorkflowTableProps, ref) => {
     }
 
     // Else if Revert Chaos is set to False and revert chaos template is present in the manifest
-    else if (
-      !revertChaos &&
-      parsedYAML.spec.templates[0].steps[
-        parsedYAML.spec.templates[0].steps.length - 1
-      ][0].name === 'revert-chaos'
-    ) {
-      parsedYAML.spec.templates[0].steps.pop(); // Remove the last step -> Revert Chaos
+    // else if (
```

Comment:
Can we remove the comments?

Reply:
Yes sure, missed this one. Have removed this in the latest commit.